WooCommerce/Classes/Extensions/UIView+SafeAreaConstraints.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+extension UIView {
```

Comment:
I'm a bit confused, I can see [this file](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Extensions/UIView+SafeAreaConstraints.swift) and [`UIView+SuperviewConstraints` in `trunk`](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Extensions/UIView+SuperviewConstraints.swift). Any reasons this PR includes these two files again?

Reply:
I realised that we'd want these fixes regardless, so separately PRed them to trunk. The merge is clean, we don't end up with duplicates. They were merged in #5653.

Sorry for the confusion, and good spot!